tests/playwright/README.md

```diff
@@ -1,146 +1,103 @@
 
-export async function teardown() {
-  if (!teardownCalled) {
-    // here comes teardown logic
-    teardownCalled = true;
+```
+  "scripts": {
+    "test:e2e": "PODMAN_DESKTOP_ARGS='/path/to/podman-desktop' playwright test tests/src/ --workers=1 --no-parallel",
   }
-}
 ```
 
-### Vitest Config
+7. Now you can implement your E2E tests in the `tests` folder of YOUR repo
+8. Execute `npm test:e2e` in order to run them
+9. When the tests finish you will find the test artifacts and logs available under `./tests/output`
 
-Example of the vitest test configuration file.
+## Usage of @podman-desktop/tests-playwright in an external repository
 
-```
-const config = {
-  test: {
-    globals: true,
-    // or use one shipped with `@podman-desktop/tests-playwright`
-    // globalSetup: './node_modules/@podman-desktop/tests/playwright/globalSetup/global-setup.ts',
-    globalSetup: './path/to/globalSetup/global-setup.ts',
-    setupFiles: './path/to/hooks/custom-extended-hooks.ts',
-    /**
-     * By default, vitest search test files in all packages.
-     * For e2e tests have sense search only is project root tests folder
-     */
-    include: ['**/tests/src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-    exclude: [
-      '**/builtin/**',
-      '**/node_modules/**',
-      '**/dist/**',
-      '**/.{idea,git,cache,output,temp,cdix}/**',
-      '**/{.electron-builder,babel,changelog,docusaurus,jest,postcss,rollup,svelte,tailwind,vite,vitest*,webpack}.config.*',
-    ],
-
-    /**
-     * A default timeout of 5000ms is sometimes not enough for playwright.
-     */
-    testTimeout: 60_000,
-    hookTimeout: 120_000,
-    // test reporters - default for all and junit for CI
-    reporters: process.env.CI ? ['default', 'junit'] : ['verbose'],
-    outputFile: process.env.CI ? { junit: 'tests/output/junit-results.xml' } : {},
-  },
-};
+This section explains how to add the npm package to a repository external to podman-desktop.
 
-export default config;
-```
+Steps:
 
-### Setting and Running the E2E tests with @podman-desktop/tests-playwright npm package
+1. Get into the Podman Desktop repo folder (should be named `podman-desktop`)
+2. Install its local dependencies by executing `pnpm install`
+3. Implement your changes to the E2E tests library (optional)
+4. Build the application with `pnpm build`
+5. Build the application's tests with `pnpm test:e2e:build`
+6. Set an environment variable by executing: `export PODMAN_DESKTOP_ARGS="/complete/path/to/podman-desktop-repo"` (the current directory) [¹]
+7. Get into YOUR repository and update the `package.json` file to have the following contents:
 
-Since you have your tests and testing framework at place, you can now run your tests from the repository.
+- Under `devDependencies`:
+  - `"@podman-desktop/tests-playwright": "next",`[²]
```

Comment:
I would mention that we also need (if we want actual latest state of `tests-playwright`, that the user needs to execute `pnpm add -D @podman-desktop/tests-playwright@next`) - it can be part of [2] note which looks good place for such info.

Reply:
you can try on your own. Checkout some extension, look at version of `tests-playwright` ion pnpm-lock.yaml, run `pnpm install`, check version again.

Reply:
So I tried with the bootc extension. In the first case this is what the the lock file contained after `pnpm install`:
```
      '@podman-desktop/tests-playwright':
        specifier: next
        version: 1.15.0-202411200746-b9b11a181a
```
And after executing `pnpm add -D @podman-desktop/tests-playwright@next`:
```
      '@podman-desktop/tests-playwright':
        specifier: 1.15.0-202411200746-b9b11a181a
        version: 1.15.0-202411200746-b9b11a181a
```
It required adding the `-w` flag because of this error though:
```
ERR_PNPM_ADDING_TO_ROOT  Running this command will add the dependency to the workspace root, which might not be what you want - if you really meant it, make it explicit by running this command again with the -w flag (or --workspace-root). If you don't want to see this warning anymore, you may set the ignore-workspace-root-check setting to true.
```
Is this what we are actually looking for?